### ISSUE TITLE
How to remove the effect of backgroud??

### ISSUE THREAD

**Issue by lily10086** [DeepSegment/ConvNet-Segment#9]
When I read your code ,The loss it tf.reduce_mean(tf.nn.sparse~~),The image‘s background also included,How to remove it？？

**Comment by txzhao** [comment]
Hi,

Sorry for such a late reply. I've been quite occupied with other projects recently... Anyway, to my knowledge, this issue should be easy to fix. You can either choose to set the label of background to -1 to ignore it when computing loss, or use tf.losses.compute_weighted_loss in front of tf.nn.sparse~. For more details, you can turn to **TheRevanchist** and **JimmyCai91**'s comments in the following link:
https://github.com/keras-team/keras/issues/6118

Cheers,
Tianxiao
